src/bundle/ui-dev/src/modules/universal-discovery/hooks/useSearchByQueryFetch.js

```diff
@@ -72,12 +75,21 @@ export const useSearchByQueryFetch = () => {
             const isImageCriterionDataEmpty = !imageCriterionData || Object.keys(imageCriterionData).length === 0;
 
             if (!isImageCriterionDataEmpty) {
-                const imageCriterion = {
-                    fieldDefIdentifier: 'image',
-                    ...imageCriterionData,
+                const imagesCriterion = damWidgetConfig.image.fieldDefinitionIdentifiers.reduce((criterions, fieldDefinitionIdentifier) => {
+                    criterions = [
```

Comment:
I think that our eslint restricts reassignment of the function parameter value. 🤔